mintpy/tsview.py

```diff
@@ -736,8 +736,7 @@ def __init__(self, cmd=None, iargs=None):
 
 
     def configure(self):
-        inps = cmd_line_parse(self.iargs)
-        inps, self.atr = read_init_info(inps)
+        inps, self.atr = read_init_info(self.iargs)
         # copy inps to self object
         for key, value in inps.__dict__.items():
             setattr(self, key, value)
```

Comment:
The `inps` Namespace object is only generated here to be saved as member variables in the `self` object. Thus, it seems simpler to me to keep the parsing and saving next to each other.

>  so that help and usage can be printed (and exit) gracefully if iargs is None

I don't think I get what do you mean.

Reply:
Addressed in comment above